stock_storage_type/tests/test_storage_type_move.py

```diff
@@ -38,28 +38,33 @@ def _test_confirmed_move(self, product=None):
         return move_to_assign
 
     def test_not_only_empty_confirmed_move(self):
-        self.pallets_location_storage_type.write({"allow_new_product": "mixed"})
+        category = self.pallets_location_storage_type.storage_category_id
+        category.allow_new_product_ids.allow_new_product = "mixed"
```

Comment:
Was a bit brutal with this change, this doesn't really comply with what we are testing, need to fix that:

- set the "Allow new product" policy through a rule, not globally on the storage category (especially true for tests on cardboxes with different lots below in this file)